site/docs/v1/tech/samlv2/google.adoc

```diff
@@ -34,14 +34,14 @@ image::applications-list-view-dialog.png[Application view dialog,width=1200,role
 
 Here are the settings you need to copy and paste:
```

Comment:
Should we mention these values will be found in the `SAML v2 Integration details` section of the dialog?